tlspuffin/src/protocol.rs

```diff
@@ -344,7 +344,7 @@ impl ExtractKnowledge<TlsQueryMatcher> for HandshakePayload {
             data: Box::new(self.clone()),
         });
         match &self {
-            HandshakePayload::HelloRequest => {}
+            HandshakePayload::HelloRequest | HandshakePayload::HelloRetryRequest(_) => {}
```

Comment:
Why did you have to change that?

Reply:
This is required to make uni tests pass for BoringSSL and WolfSSL, these tests were not performed before this PR.